platform/qemu.go

```diff
@@ -265,19 +265,30 @@ func CreateQEMUCommand(board, uuid, biosImage, consolePath, confPath, diskImageP
 			"-cpu", "host",
 			"-m", "2048",
 		}
+	case "s390x--s390x-usr":
+		qmBinary = "qemu-system-s390x"
+		qmCmd = []string{
+			"qemu-system-s390x",
+			"-machine", "s390-ccw-virtio,accel=kvm",
+			"-cpu", "host",
+			"-m", "2048",
+		}
 	default:
 		panic("host-guest combo not supported: " + combo)
 	}
 
 	qmCmd = append(qmCmd,
-		"-bios", biosImage,
 		"-smp", "1",
 		"-uuid", uuid,
 		"-display", "none",
 		"-chardev", "file,id=log,path="+consolePath,
 		"-serial", "chardev:log",
 	)
 
+	if runtime.GOARCH != "s390x" {
+		qmCmd = append(qmCmd, "-bios", biosImage)
+	}
+
 	if isIgnition {
 		qmCmd = append(qmCmd,
 			"-fw_cfg", "name=opt/com.coreos/config,file="+confPath)
```

Comment:
Does `s390x` support `fw_cfg`? If not it's probably necessary to modify this `if` block to `isIgnition && board != "s390x"`.

Reply:
@arithx thank you to have a look. `s390x` does not support `fw_cfg`. My team was going to have separate PR with workaround for this problem, see https://github.com/coreos/mantle/issues/1045, because we still need to pass ignition file to the guest. Would you prefer for this PR just to exclude ignition config for s390x at all at this moment (as you suggested), waiting for another PR to fix `fw_cfg` first, or have all the fixes for x390x in this PR?

Reply:
@barthy1 that's fine, was just pointing it out. Feel free to do it in a separate PR.